Makefile

```diff
@@ -270,7 +270,10 @@ standalone: node_modules $(built_standalone_js_files) $(built_standalone_shared_
 	$(RSYNC) shared $(BUILT)/$@/content
 
 .PHONY: add-on
-add-on: node_modules $(built_add_on_js_files) $(built_add_on_shared_js_files) $(built_add_on_l10n_files) $(BUILT)/$(ADD-ON)/chrome.manifest $(BUILT)/add-on/chrome/content/preferences/prefs.js
+add-on: node_modules $(built_add_on_js_files) $(built_add_on_shared_js_files) \
+	      $(built_add_on_l10n_files) $(BUILT)/$(ADD-ON)/chrome.manifest \
+	      $(BUILT)/$(ADD-ON)/chrome/locale/chrome.manifest \
+	      $(BUILT)/add-on/chrome/content/preferences/prefs.js
```

Comment:
Let's make these dependencies all be on separate lines as we can't get it to fit on a single line.